library/std/src/time.rs

```diff
@@ -612,9 +593,8 @@ impl SystemTimeError {
     /// methods of [`SystemTime`] whenever the second system time represents a point later
```

Comment:
Should we change the text a bit here such that we could use `SystemTime::duration_since` and `SystemTime::elapsed` directly? Like
```
    /// A `SystemTimeError` is returned from the [`SystemTime::duration_since`] and
    /// [`SystemTime::elapsed`] whenever the second system time represents a point later
```

Reply:
That's a good idea !